base_cancel_confirm/model/base_cancel_confirm.py

```diff
@@ -67,13 +70,15 @@ def fields_view_get(
                 for new_element in new_node:
                     node.addnext(new_element)
             # Override context for postprocessing
-            View = self.env["ir.ui.view"]
-            if view_id and res.get("base_model", self._name) != self._name:
-                View = View.with_context(base_model_name=res["base_model"])
-            new_arch, new_fields = View.postprocess_and_fields(doc, self._name)
+            if view_id:
+                view = self.env["ir.ui.view"].search([("id", "=", view_id)])
```

Comment:
Reassigning 'self.env["ir.ui.view"]' to the View variable makes more sense to me.